src/components/PickupBox/PickupBox.jsx

```diff
@@ -0,0 +1,172 @@
+import { React, useState } from 'react';
+import {
+  Accordion,
+  AccordionItem,
+  AccordionPanel,
+  AccordionButton,
+  AccordionIcon,
+  ChakraProvider,
+  FormControl,
+  FormLabel,
+  Input,
+  Textarea,
+} from '@chakra-ui/react';
+
+import PropTypes from 'prop-types';
+import styles from './PickupBox.module.css';
+import RejectBoxPopup from '../AlertPopups/RejectBoxPopup/RejectBoxPopup';
+import ApproveBoxIcon from '../BoxIcons/ApproveBoxIcon.svg';
+import RejectBoxIcon from '../BoxIcons/RejectBoxIcon.svg';
+import PickupBoxIcon from '../BoxIcons/PickupBoxIcon.svg';
+import ImageVector from '../BoxIcons/ImageVector.svg';
```

Comment:
Can we try to use react or chakra icons as much as possible instead of SVGs?
https://chakra-ui.com/docs/media-and-icons/icon
https://react-icons.github.io/react-icons/